src/UglyToad.PdfPig.Fonts/TrueType/Tables/CMapSubTables/Format4CMapTable.cs

```diff
@@ -56,6 +57,58 @@ public int CharacterCodeToGlyphIndex(int characterCode)
             return 0;
         }
 
+        public int[] GetCharactersCode()
+        {
+            var charactersCode = new List<int>();
+
+            ushort minCharacterStart = 0xffff;
+
+            foreach (var segmentIndex in Enumerable.Range(0, Segments.Count - 1))
+            {
+                var segment = Segments[segmentIndex];
+
+                minCharacterStart = Math.Min(minCharacterStart, (ushort)segment.StartCode);
+
+                var characterCountInSegment = (segment.EndCode - segment.StartCode) + 1;
+                if (segment.IdRangeOffset == 0)
+                {
+                    var chars = Enumerable.Range(segment.StartCode, characterCountInSegment);
+                    charactersCode.AddRange(chars);
+                }
+                else
+                {
+                    var chars = Enumerable.Range(0, characterCountInSegment).Where(i =>
+                    {
+                        var offset = segment.IdRangeOffset / 2 + i;
```

Comment:
This filter (`Where` Clause) it's unnecessary. But, since I'm really no expert in TrueType fonts it gives me a little more security having it there.